arm/Microsoft.Automation/automationAccounts/.parameters/parameters.json

```diff
@@ -46,6 +46,36 @@
                 }
             ]
         },
+        "variables": {
+            "value": [
+                {
+                    "name": "TestString",
+                    "value": "\"TestString\"",
+                    "description": "TestStringDescription"
+                },
+                {
+                    "name": "TestInteger",
+                    "value": "500",
+                    "description": "TestIntegerDescription"
+                },
+                {
+                    "name": "TestBoolean",
+                    "value": "false",
+                    "description": "TestBooleanDescription"
+                },
+                {
+                    "name": "TestDateTime",
+                    "value": "\"\\/Date(1637934042656)\\/\"",
+                    "description": "TestDateTimeDescription"
+                },
+                {
+                    "name": "TestEncryptedVariable",
+                    "value": "\"TestEncryptedValue\"",
+                    "description": "TestEncryptedDescription",
+                    "isEncrypted": true
+                }
+            ]
+        },
         // "linkedWorkspaceId": {
```

Comment:
Let's link the adp workspace, or?

Reply:
Yes that parameter is used for that. However I commented them to be reused once we'll be set with the removal of those resources. Let's keep them there for now